wkcuber/api/Layer.py

```diff
@@ -199,30 +199,36 @@ def _pad_existing_mags_for_downsampling(
 
     def downsample(
         self,
-        from_mag: Mag,
-        max_mag: Optional[Mag],
-        interpolation_mode: str,
-        compress: bool,
+        from_mag: Optional[Mag] = None,
+        max_mag: Optional[Mag] = None,
+        interpolation_mode: str = "default",
+        compress: bool = True,
         anisotropic: Optional[bool] = None,
         scale: Optional[Tuple[float, float, float]] = None,
         buffer_edge_len: Optional[int] = None,
         args: Optional[Namespace] = None,
     ) -> None:
+        if from_mag is None:
+            assert (
+                len(self.mags.keys()) > 0
+            ), "Failed to downsample data. The mag exists that could be used as 'from_mag'"
+            from_mag = Mag(min(self.mags.keys()))
```

Comment:
Why `min`? Shouldn't it be `max` to get the `Highest currently available mag` mentioned in #298.